### PR TITLE
feat(devices): enable devices for sampled users

### PR DESCRIPTION
**Pull request by vladikoff**
@vbudhram r?

Works the same way as https://github.com/mozilla/fxa-content-experiments/blob/dev/verification/mailcheck.js#L11

### CODE REVIEW

**Comment by vladikoff** [comment]
Value comes from : https://github.com/mozilla/fxa-content-server/blob/8ffa42ff634a9c9eaa85c77072a18cfc8ea7102d/app/scripts/views/settings/clients.js#L87

File: clients/devices.js [hunk 1]
```diff
@@ -3,11 +3,13 @@
 module.exports = {
   name: 'Is the devices list settings section visible',
   startDate: '2015-01-01',
-  subjectAttributes: ['forceDeviceList'],
+  subjectAttributes: ['forceDeviceList', 'isMetricsEnabledValue'],
```
Comment (vbudhram) [inline clients/devices.js:6]: Does this need `uniqueUserId` similar to [mailcheck](https://github.com/mozilla/fxa-content-experiments/blob/dev/verification/mailcheck.js#L7)?

**Review by vbudhram — COMMENTED** [review]
@vladikoff Just the one q, running this locally and pointing to this branch shows the devices view. Is it possible to test being excluded from the experiment, devices view always showed up for me, even when trying different users.

File: clients/devices.js (hunk 1 above)
Comment (vladikoff) [reply clients/devices.js:6]: nope, we are not splitting this into groups so we don't need `uniqueUserId`

Comment (vbudhram) [reply clients/devices.js:6]: Heh, guess that kind of answers my top level question too :)

**Review by vbudhram — APPROVED** [review]

**Comment by vladikoff** [comment]
@vbudhram in dev all users are 'isSampledUser' : https://github.com/mozilla/fxa-content-server/blob/64081c62a3665e1a6ffb064b2011307ee0dd1332/app/scripts/lib/app-start.js#L184 

It's based on your config: https://github.com/mozilla/fxa-content-experiments/blob/dev/metrics/is-sampled-user.js#L9

**Comment by vladikoff** [comment]
This commit will be cherry picked onto train-70 branch to enable devices on October 3rd or when train-70 deploys.

**Pull request merged**